app/views/organizations/_disabled_organization_select.html.erb

```diff
@@ -1,5 +1,5 @@
 <% aria_label = org[:role] != 'admin' ? t('views.organizations.you_are_not_owner') : t('views.organizations.already_added', owner: org[:owner_login]) %>
-<div class="organization-select-target disabled tooltipped tooltipped-s" aria-label="<%= aria_label %>">
-  <%= image_tag "https://avatars.githubusercontent.com/u/#{org[:github_id]}?v=3&size=200", class: 'avatar organization-select-avatar', width: 100, height: 100, alt: "@#{org[:login]}" %>
-  <span class="organization css-truncate css-truncate-target" title=<%= "@#{org[:login]}" %>><%= "@#{org[:login]}" %></span>
+<div class="d-block width-full text-center hover-grow border rounded-2 box-shadow-medium bg-gray p-3 tooltipped tooltipped-s" aria-label="<%= aria_label %>">
```

Comment:
Since we're replacing a lot of existing classes with primer, is it possible to delete the legacy CSS? 

This can be in a follow-up PR. Ideally we'd also introduce some CI that finds unused CSS and unused classes in the markup.

Reply:
@sophshep That's a great idea! I've been avoiding removing some of the old classes because they're used in other parts in the application but once we remove all of the uses and verify that the app still looks fine we can add CI to detect and remove the unused CSS.